launchers/junit/src/testFixtures/java/org/eclipse/dataspaceconnector/junit/launcher/MockVault.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.junit.launcher;
+
+import org.eclipse.dataspaceconnector.spi.security.Vault;
+import org.eclipse.dataspaceconnector.spi.security.VaultResponse;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * An in-memory vault for testing.
+ */
+public class MockVault implements Vault {
```

Comment:
This class seems not to be used anywhere. There is another `MockVault` in `DemoProtocolsTransferExtensionTest.java`, which looks pretty much the same.

Reply:
Oh this looks like a rebase problem. I'll delete it